project/tasks/get_started.rst

```diff
@@ -0,0 +1,84 @@
+======================
+Get Started with Tasks
+======================
+
+Break down a project by easily creating, assigning and organizing tasks. Set priorities to help you
+determine where you want to invest your time first, manage files within it and keep everything you
+need to complete your work in one place. Ensure that your strategies are turned into actionable
+plans, and as a result, have a successful project.
+
+Create a task
+=============
+
+Click on *Create*. Once created, add more details opening it and clicking on *Edit*.
+
+.. image:: media/create.png
+   :align: center
+   :height: 300
+   :alt: Click on create under a project to initiate a task in Odoo Project
+
+Assignation
+-----------
+
+Choose the responsible person for a task under *Assigned to*.
+
+.. image:: media/assignation.png
+   :align: center
+   :alt: Choosing the responsible person for the task in Odoo Project
+
+Sort tasks by priority
+======================
+
+| Mark the star on your task to set it as a high priority one. That automatically moves the task
+  to the top of the column. Tasks that are not starred are classified according to their
+  deadlines.
+| Note that tasks with dates passed their deadlines will be shown in red; tasks with a deadline for
+  the current date are orange.
+
+.. image:: media/priority.png
+   :align: center
+   :alt: Task prioritized are shown first in the list in Odoo Project
+
+Manage files in tasks
+=====================
+
+Add images/documents to your task by clicking on the *Attachment icon*.
+
+.. image:: media/manage_files.png
+   :align: center
+   :alt: Click on the attachment icon to manage files in Odoo Project
+
+.. tip::
+   Files from emails are automatically saved.
+
+Set cover images in tasks
+=========================
+
+*Set Cover Image* to your task and quickly get to comprehend what the task is about. The image is
```

Comment:
```suggestion
*Set Cover Images* to your task and quickly get to comprehend what the task is about. The image is
```

Reply:
As I'm relating to the name of the feature in the application, I'll keep it singular.

Reply:
Then, I would say "Use *Set Cover Image*" so no misunderstanding :)